src/main/java/com/amazon/ion/impl/macro/MacroEvaluator.kt

```diff
@@ -18,7 +18,7 @@ import com.amazon.ion.impl.macro.Expression.*
  *  - Call [stepOut] to step out of the current container.
  */
 class MacroEvaluator(
-    private val encodingContext: EncodingContext,
+    val encodingContext: EncodingContext,
```

Comment:
I needed to be able to access the encoding context in tests. I'm open to better ways of allowing that.